docs/application/native/guides/device/system.md

```diff
@@ -159,6 +159,16 @@ The following table lists the consumer IR feature keys.
 |----------------------------------------|--------|------------------------------------------|
 | `http://tizen.org/feature/consumer_ir` | `bool` | The platform returns `true` for this key, if the device supports the Consumer Infrared (CIR) feature. |
 
+The following table lists the content feature keys.
+
+<a name="content"></a>
+**Table: Content feature keys**  
+
+| Key                                    | Type   | Description                              |
+|----------------------------------------|--------|------------------------------------------|
+| `http://tizen.org/feature/content.scanning.others` | `bool` | The platform returns true for this key, if the device supports the media scanning feature for OTHERS-type files which are not included in the media content types such as IMAGE, VIDEO, SOUND or MUSIC. |
```

Comment:
Suggestion:

The platform returns true for this key, if the device supports the media scanning feature for `OTHERS` type files, which are not included in the media content-types such as image, video, sound, or music.

I have few questions:
1.	I have suggested using code tag for OTHERS as we are referring to the type of files. Please let me know if it’s okay.
2. Is the content-types IMAGE, VIDEO, SOUND or MUSIC in upper case as this is how the input is taken? If not, can we replace it with image, video, sound, or music?

Reply:
1. OTHERS-type => OTHERS type is OK
2. input is in upper case like IMAGE, VIDEO.. So I think it's better not to change it.

Reply:
@haejeongkim,

Thank you for clarifying my doubts.

Please rephrase the content to:

The platform returns true for this key, if the device supports the media scanning feature for OTHERS type files, which are not included in the media content-types such as IMAGE, VIDEO, SOUND, or MUSIC.